src/components/Card/Card.tsx

```diff
@@ -41,19 +62,23 @@ export const Card = ({
   return (
     <section
       onClick={toggleSelected}
-      className={`${cardClass} ${selected ? `border-${tagColor}` : ''} `}>
+      className={`${cardClass} ${selected ? `card--${type}` : ''} `}>
       <figure className='card__container-image'>
-        <img className='card__image' src={imgSrc} alt={imgAlt} />
+        <div
+          className='card__image'
+          style={{ backgroundImage: `url(${src})` }}
+          role='img'
+          aria-label={alt}
+          title={alt}
+        />
       </figure>
       <article className='card__container'>
-        <p className={`${cardTagClass} color-${tagColor} bg-${tagBg}`}>
-          tag text
-        </p>
-        <p className='card__lead lead-text color-dark700'>0012-DE-Muller</p>
-        <p className='card__overline-text overline-text color-dark500'>
-          Projekt:
+        <p className={`${cardTagClass} card--${type}-${tagType}`}>{tagText}</p>
+        <p className='card__lead lead-text color-gray700'>{leadText}</p>
+        <p className='card__overline-text overline-text color-gray500'>
+          {overlineText}
         </p>
-        <p className='card__major-text major-text color-dark700'>Point 145</p>
+        <p className='card__major-text major-text color-gray700'>{majorText}</p>
       </article>
     </section>
```

Comment:
Tu można my na pewno nad semantyką pomyśleć zamiast wszędzie dawać `p`. Tag to raczej zwykły `span`, jakiś `hX` byś na pewno na karcie przydał.

`section` i `article` (_szczególnie to drugie_) IMO trochę przesadzone semantycznie patrząc na ten komponent. Sam bym się raczej skłaniał do zwykłego `div`a lub `aside`a - w zależności od kontekstu w jakim będzie używany ten komponent.